_posts/en/newsletters/2025-01-31-newsletter.md

```diff
@@ -0,0 +1,165 @@
+  any transactions in that block that they do not already have in their
+  mempool (or in their _extrapool_, which is a special reserve that aims
+  to help with compact block reconstruction).  This significantly slows
+  down the speed of block propagation and contributes to miner
+  centralization.
+
+  0xB10C found that the frequency of requests increases significantly as
+  the size of the mempool grows.  Several developers discussed possible
+  causes, with initial data seeming to indicate the missing transactions
+  were _orphans_---child transactions of unknown parents, which Bitcoin
+  Core only keeps around briefly in case their parents arrive within a
+  short time.  Improved tracking and requesting of the parents of orphan
+  transactions, recently merged into Bitcoin Core (see [Newsletter
+  #338][news338 orphan]), could help improve this situation.
+
+  Developers also discussed other possible solutions.  Orphan
+  transactions can't reasonably be keep for a long time because an
+  attacker can create them for free, but it might be possible to persist
+  a larger number of them and other evicted transactions for a longer
+  time in the extrapool.  The discussion was inconclusive at the time of
+  writing.
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+_New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates._
+
+FIXME:harding to update (or remove, if no releases/RCs)
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
+repo], and [BINANAs][binana repo]._
+
+- [Bitcoin Core #31376][] Miner: never create a template which exploits the timewarp bug
+
+- [Bitcoin Core #31583][] rpc: add target to getmininginfo field and show next block info
+
+- [Bitcoin Core #31590][] descriptors: Try pubkeys of both parities when retrieving the private keys for an xonly pubkey in a descriptor
```

Comment:
```suggestion
- [Bitcoin Core #31590][] descriptors: Try pubkeys of both parties when retrieving the private keys for an xonly pubkey in a descriptor
```

Reply:
Parity seems plausible in the context of x-only keys